app/components/web-components/vl-datepicker.hbs

```diff
@@ -43,6 +41,13 @@
       @shorthandCurrentMonth={{true}}
       @disabled={{@disabled}}
     />
-    <Auk::Icon @name="calendar" />
-  </div>
-{{/if}}
\ No newline at end of file
+  {{/if}}
+  {{#if @enableClear}}
+    <Auk::Button
+      @skin="muted-borderless"
+      @layout="icon-only"
+      @icon="x"
+      {{on "click" @onClear}}
+    />
+  {{/if}}
```

Comment:
@MikiDi Misschien hier ook even kort duiding: de clear knop die het vl-datepicker component wat uitbreidde heb ik binnen het component zelf gebracht om zo de `auk-input-with-icon` class er uit te kunnen refactoren en proper binnen `au-kaleidos-css` brengen.

Reply:
@MikiDi Binnenkort zou die legacy vl-versie hiervan willen er uit halen, maar ik hou sowieso in mijn achterhoofd dat deze functionaliteit ook binnen de auk-versie zou moeten komen?